vcluster/manage/backup-restore/volume-snapshots/aws-ebs-guide.mdx

```diff
@@ -0,0 +1,334 @@
+---
+title: AWS-EBS guide
+sidebar_label: AWS-EBS guide
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TenancySupport from '../../../_fragments/tenancy-support.mdx';
+import Mark from "@site/src/components/Mark";
+import InstallCli from '../../../_partials/deploy/install-cli.mdx';
+import KubeconfigUpdate from '@site/docs/_partials/kubeconfig_update.mdx';
+
+This guide walks you through creating volume snapshots for a vCluster with persistent data and restoring that data from the snapshot. You'll deploy a sample application that writes data to a persistent volume, create a snapshot, simulate data loss, and restore from the snapshot using AWS EBS as the storage provider.
+
+:::info Supported CSI Drivers
+vCluster officially supports volume snapshots with **AWS EBS CSI Driver** and **OpenEBS**. This walkthrough demonstrates the complete end-to-end process using AWS as an example. Similar steps can be adapted for other supported CSI drivers.
+:::
+
+## Prerequisites
+
+Before starting, ensure you have:
+
+- An existing Amazon EKS cluster with the EBS CSI Driver installed. Follow the [EKS deployment guide](../../../../deploy/control-plane/container/environment/eks) to set up your cluster.
+- The vCluster CLI installed
+- Completed the [volume snapshots setup](./#setup) based on your chosen tenancy model
+- Configured the default VolumeSnapshotClass for the EBS CSI driver
```

Comment:
Would be nice to have more info here. Either linking out, or showing an example below.